docs/content/docs/forwarder.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "Transactional publishing of messages"
+description = "Publishing messages in transactional way with help of Forwarder component"
+date = 2021-01-13T12:47:30+01:00
+weight = -400
+draft = false
+bref = "Publishing messages in transactions, while emitting events along with storing data in a database"
+toc = true
++++
+
+## Publishing messages in transactions (and why we should care) 
+While working with an event-driven application, you may in some point need to store an application state and publish a message 
```

Comment:
It may be good to add tl;dr a problem statement in the first paragraph. Just to ensure that reader will instantly know what is the problem. If he will not find it in the first paragraph, it's a high chance that he will not read it further. Later we can go into the details and example :wink: 

It may be worth mentioning that we are avoiding _two phrase commit_ + some diagram. (bonus point for diagram in `.svg`).

Reply:
I made the first paragraph focused on a problem and its solution - the Forwarder. :P svg diagram explaining how the component works added as well.